test/snippet/alphabet/cigar/cigar_get_index.cpp

```diff
@@ -0,0 +1,18 @@
+#include <seqan3/alphabet/cigar/cigar.hpp>
+#include <seqan3/alphabet/cigar/cigar_op.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    using seqan3::get;
+    using seqan3::operator""_cigar_op;
+
+    seqan3::cigar letter0{10, 'M'_cigar_op};
```

Comment:
```suggestion
    seqan3::cigar letter{10, 'M'_cigar_op};
```

there is no other letter.